src/main/kotlin/sc/iview/commands/edit/Properties.kt

```diff
@@ -468,6 +491,10 @@ class Properties : InteractiveCommand() {
             maybeRemoveInput("slicedVolumes", VolumeSelectorWidget.VolumeSelection::class.java)
         }
 
+        if (node is Line){
```

Comment:
Oh good, Kotlin decided `instanceof` &rarr; `is`, huh?

| Concept | Java | Python | Kotlin |
|---------|------|--------|--------|
| referential equality | `==` | `is` | `===` |
| structural equality | `equals` | `==` | `==` |
| type equality | `instanceof` | `isinstance` | `is` |

![](https://venturebeat.com/wp-content/uploads/2018/01/babaisyou.gif)